test/common/chromium-cdp-driver.js

```diff
@@ -254,17 +254,9 @@ function setupLocalFunctions(client) {
 }
 
 // helper functions for testlib.py which are too unwieldy to be poked in from Python
-function getFrameExecId(frame) { // eslint-disable-line no-unused-vars
-    if (frame === null)
-        frame = "cockpit1";
-    const frameId = frameNameToFrameId[frame];
-    if (!frameId)
-        return -1;
-    const execId = frameIdToContextId[frameId];
-    if (!execId)
-        return -1;
-    return execId;
-}
+
+// eslint-disable-next-line no-unused-vars
```

Comment:
Is this eslint hint still necessary?

Reply:
Yes, it replaces the old `eslint-disable-line` as otherwise the line would get too long. `getFrameExecId()` is not called within this file, it's an API for `cdp.py`. But we can' t mark it as `export` as this isn't a proper ES6 module.

Reply:
I see, thanks!